plugins/pagerduty/README.md

```diff
@@ -95,6 +95,30 @@ annotations:
   pagerduty.com/integration-key: [INTEGRATION_KEY]
 ```
 
+### The homepage component
+
+You may also add the component to the homepage of Backstage. Edit the `HomePage.tsx` file, import `PagerDutyHomepageCard` and add it to the home page. e.g.
+
+```typescript jsx
+...
+export const homePage = (
+  <Page themeId="home">
+    ...
+    <Content>
+      <CustomHomepageGrid config={defaultConfig}>
+        ...
+        <PagerDutyHomepageCard
+          name="My Service"
```

Comment:
Are these something the user would like to change on their own? The `createCardExtension` supports also settings schema that allows users to specify their own settings per widget in the grid using JSON Schema Form. See for example the RandomJoke component as a simple example or Q&A plugin here: https://github.com/drodil/backstage-plugin-qeta/blob/main/plugins/qeta/src/plugin.ts#L45